src/api/app/controllers/webui/request_controller.rb

```diff
@@ -30,13 +28,20 @@ class Webui::RequestController < Webui::WebuiController
                                   if: -> { Flipper.enabled?(:request_show_redesign, User.session) }
   before_action :check_beta_user_redirect, only: %i[beta_show build_results rpm_lint changes mentioned_issues]
   before_action :redirect_to_tasks, only: [:index], unless: -> { Flipper.enabled?(:request_index, User.session) }
-  before_action :set_requests, :set_filter_involvement, :set_filter_state, :set_filter_action_type, :set_filter_creators,
-                :filter_requests, :set_selected_filter, only: [:index], if: lambda {
-                                                                              Flipper.enabled?(:request_index, User.session)
-                                                                            }
+
   after_action :verify_authorized, only: [:create]
 
   def index
+    if Flipper.enabled?(:request_index, User.session)
```

Comment:
Is there even an index action outside of beta? :)

Reply:
Nop, there isn't any. Removed. :+1: